marklogic-data-hub/src/test/ml-config/security/roles/data-hub-test-internal.json

```diff
@@ -0,0 +1,56 @@
+{
+  "role-name": "data-hub-test-internal",
+  "description": "Used to test different roles/privileges at the server level",
+  "privilege": [
+    {
+      "privilege-name": "privilege-add-roles",
+      "action": "http://marklogic.com/xdmp/privileges/privilege-add-roles",
+      "kind": "execute"
+    },
+    {
+      "privilege-name": "grant-my-roles",
+      "action": "http://marklogic.com/xdmp/privileges/grant-my-roles",
+      "kind": "execute"
+    },
+    {
+      "privilege-name": "grant-all-roles",
+      "action": "http://marklogic.com/xdmp/privileges/grant-all-roles",
+      "kind": "execute"
+    },
+    {
+      "privilege-name": "get-role",
+      "action": "http://marklogic.com/xdmp/privileges/get-role",
+      "kind": "execute"
+    },
+    {
+      "privilege-name": "create-role",
+      "action": "http://marklogic.com/xdmp/privileges/create-role",
+      "kind": "execute"
+    },
+    {
+      "privilege-name": "remove-role",
+      "action": "http://marklogic.com/xdmp/privileges/remove-role",
+      "kind": "execute"
+    },
+    {
+      "privilege-name": "get-user",
+      "action": "http://marklogic.com/xdmp/privileges/get-user",
+      "kind": "execute"
+    },
+    {
+      "privilege-name": "create-user",
```

Comment:
will this and other *-user privileges impact running of tests in DHS?

Reply:
It is only used as part of an amp on a testing library function. I'm not aware of any negative impact of using this in DHS.